test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -277,13 +277,16 @@ Date: REGEX(.*)
 03. GET /v2/entities
```

Comment:
I think we will be supporting v1 and v2 simultaneously for a very long time and that it is only good to have functests that test that they actually can work together.

I would definitely not remove any already implemented functests. Instead I would copy all those 'tests that were "meant for v2" but as POST /v2/entities didn't exist they use v1 instead'
and make the copy use 100% v2